tests fail in `WINO=1 python -m pytest test/test_ops.py -k conv`

```
FAILED test/test_ops.py::TestOps::test_fancy_conv2d - AssertionError: invalid shrink ((0, 2), (0, 3), (0, 9), (0, 26)) for (2, 3, 8, 28)
FAILED test/test_ops.py::TestOps::test_nested_conv2d - Exception: backward pass tensor 0 failed shape (1, 32, 9, 9): 
FAILED test/test_ops.py::TestOps::test_output_padded_conv_transpose2d - AssertionError: invalid shrink ((0, 2), (0, 4), (0, 14), (0, 16)) for (2, 4, 12, 16)
FAILED test/test_ops.py::TestOps::test_padded_conv2d_bs1 - AssertionError: invalid shrink ((0, 1), (0, 4), (0, 11), (0, 28)) for (1, 4, 8, 28)
FAILED test/test_ops.py::TestOps::test_padded_conv2d_p21 - AssertionError: invalid shrink ((0, 4), (0, 4), (0, 13), (0, 28)) for (4, 4, 12, 28)
FAILED test/test_ops.py::TestOps::test_padded_conv2d_p22 - AssertionError: invalid shrink ((0, 4), (0, 4), (0, 13), (0, 30)) for (4, 4, 12, 32)
```

--- a/test/unit/test_winograd.py
+++ b/test/unit/test_winograd.py
@@ -73,5 +73,12 @@ class TestWinograd(unittest.TestCase):
     x,w = Tensor.empty(1,IC,Y,X,dtype=dtypes.half), Tensor.empty(OC,IC,3,3,dtype=dtypes.half)
     self.assertEqual(Tensor.conv2d(x,w).dtype, dtypes.half)
 
+  def test_padded_conv2d(self):
+    # tests padding order in winograd
+    x,w = Tensor.rand(1,3,11,28).realize(), Tensor.rand(4,3,3,3).realize()
+    with Context(WINO=0): expected = Tensor.conv2d(x,w,padding=1).realize()
+    with Context(WINO=1): result = Tensor.conv2d(x,w,padding=1).realize()
+    np.testing.assert_allclose(result.numpy(), expected.numpy(), atol=1e-4)
+
 if __name__ == '__main__':
   unittest.main(verbosity=2)

--- a/tinygrad/tensor.py
+++ b/tinygrad/tensor.py
@@ -2338,7 +2338,7 @@ class Tensor(OpMixin):
     # TODO: stride == dilation
     # use padding to round up to 4x4 output tiles
     # (bs, cin_, tyx, HWI)
-    pads = [[padding_[i*2], padding_[i*2+1] + (-(dim + sum(padding_[i * 2:(i + 1) * 2]) - 2) % 4)] for i, dim in enumerate(self.shape[-len(HW):])]
+    pads = [[padding_[i*2], padding_[i*2+1] + (-(dim+sum(padding_[i*2:(i+1)*2])-2) % 4)] for i, dim in enumerate(reversed(self.shape[-len(HW):]))]
     d = self.pad(sum(pads, []))._pool(HWI, HWO)
     # move HW to the front: # (HWI, bs, cin_, tyx)
     d = d.permute(*range(len(d.shape)-len(HW),len(d.shape)), *range(len(d.shape)-len(HW)))

